template/manifest.json.tera

```diff
@@ -1,15 +1,14 @@
 ---
 whiskers:
-  version: "2.0.2"
+  version: "2.1.0"
   matrix:
     - flavor
     - accent
   filename: "themes/{{flavor.identifier}}_{{accent}}/manifest.json"
-accent: "mauve"
 ---
 {
-  "name": "Catppuccin for DevTools {{flavor.name | capitalize}} {{accent | capitalize}}",
-  "description": "Soothing pastel theme for DevTools - {{flavor.name | capitalize}} {{accent | capitalize}}",
```

Comment:
Updated the title to include "Chrome", `flavor.name` is capitalized by default so no need.